vue-components/src/components/ToggleButtonGroup.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<span class="wikit wikit-ToggleButtonGroup">
+		<slot />
+	</span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export interface ToggleButtonGroupInjection {
+	groupValue: ( () => string ) | null;
+	toggleListener: ( ( value: string ) => void ) | null;
+}
+
+export default Vue.extend( {
+	name: 'ToggleButtonGroup',
+	provide(): ToggleButtonGroupInjection {
+		return {
+			groupValue: (): string => this.value,
+			toggleListener: ( event: string ): void => {
+				this.$emit( 'input', event );
+			},
+		};
+	},
+	props: {
+		value: {
+			required: true,
+			type: String,
+		},
+	},
+} );
+</script>
+<style lang="scss">
+.wikit-ToggleButtonGroup :not(:last-child) {
+	border-inline-end: none;
+	border-start-end-radius: 0;
+	border-end-end-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child) {
+	border-start-start-radius: 0;
+	border-end-start-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child) {
+	border-start-start-radius: 0;
+	border-end-start-radius: 0;
+}
+
+.wikit-ToggleButtonGroup :disabled {
+	border-block: none;
+}
+
+.wikit-ToggleButtonGroup :focus {
+	outline: none;
+}
+
+.wikit-ToggleButtonGroup :not(:first-child):disabled {
+	border-inline-start-color: $color-base-100;
+}
+
+.wikit-ToggleButtonGroup {
+	[dir='ltr'] & :not(:first-child) {
+		border-top-left-radius: 0;
+		border-bottom-left-radius: 0;
+	}
+
+	[dir='rtl'] & :not(:first-child) {
+		border-top-right-radius: 0;
+		border-bottom-right-radius: 0;
+	}
+
+	[dir='ltr'] & :not(:last-child) {
+		border-top-right-radius: 0;
+		border-bottom-right-radius: 0;
+	}
+
+	[dir='rtl'] & :not(:last-child) {
+		border-top-left-radius: 0;
+		border-bottom-left-radius: 0;
+	}
```

Comment:
It turns out these are not actually needed anymore, the problem was Storybook itself, as @Ladsgroup  mentioned in #358, which means it is not a problem with logical-css after all, and is unrelated to https://github.com/csstools/postcss-logical/issues/27

Reply:
Oh thanks. Removed now.